### ISSUE TITLE
Menu item to select the default currency symbol

### ISSUE THREAD

**Issue by andrewtavis** [scribe-org/Scribe-iOS#314]
### Terms

- [X] I have searched [open and closed feature requests](https://github.com/scribe-org/Scribe-iOS/issues?q=is%3Aissue+label%3Afeature)
- [X] I agree to follow Scribe-iOS' [Code of Conduct](https://github.com/scribe-org/Scribe-iOS/blob/main/.github/CODE_OF_CONDUCT.md)

### Description

This issue would create a menu item whereby the user would be taken to a separate screen to select a default currency symbol for the keyboard that appears on the `123` keys. The process is detailed in the [designs in Figma](https://www.figma.com/file/c8945w2iyoPYVhsqW7vRn6/scribe_public_designs?type=design&node-id=405%3A393&t=nCPFM9zwswgPWWI6-1). The option would be in general keyboard settings and for those on a per-keyboard basis.

- Blocked by #16 and #309

### Contribution

This issue is a part of Google Summer of Code 2023 😊 Happy to assist as needed!

**Comment by andrewtavis** [comment]
Place that this is shown in the Figma is: https://www.figma.com/file/c8945w2iyoPYVhsqW7vRn6/scribe_public_designs?type=design&node-id=513%3A1616&mode=design&t=dh3jyNuvsEhKhmR2-1

**Comment by lillian-mo** [comment]
I would be happy to work on this issue :)

**Comment by vickcoo** [comment]
Hello @andrewtavis 👋🏻, I would like try to work on this.

**Comment by andrewtavis** [comment]
If all sounds good, feel free to write in #238 and I'll assign that one too! Then we can work to get both of these done! For that one the question is how to make the currency symbol on the number keys variable and replace it with a different one while also updating those that are on the symbol keys :)

**Comment by andrewtavis** [comment]
No worries whatsoever, @vickcoo! At least on my end I'm understanding you well, and happy to explain things as needed to make things clear 😊

What I'm considering is:
- You're right that we need to merge #476 before this one as it has code that we need
- The differences between this issue and #238 are:
    - This issue is to add the menu option to select the currency symbol (change is within [Scribe-iOS/Scribe](https://github.com/scribe-org/Scribe-iOS/tree/main/Scribe), the application's menu)
    - #238 is to reflect the change of the currency symbol in the keyboard UI (change is within [Scribe-iOS/Keyboards](https://github.com/scribe-org/Scribe-iOS/tree/main/Keyboards), the change on the keyboards based on the menu selection here)

What you could do is look into the places that would need the change in #238 and we could discuss them over there? Let me head over and document it a bit for you :)

**Comment by vickcoo** [comment]
Thanks for your patience and explanation @andrewtavis ! I got it. Let's discuss it further over there, Here's a summary of the steps as I understand them
1. solve the #238
2. wait for #476 merged
3. solve this issue

**Comment by andrewtavis** [comment]
Yes you can definitely solve #238 in the meantime, and the other PR should be in by the end of the week 😊 Thanks, @vickcoo!

**Comment by andrewtavis** [comment]
Hey @vickcoo 👋 The fall has been quite busy for other projects, but we finally got #476 merged in :) Would you have interest in working on this now?

**Comment by andrewtavis** [comment]
Nice! Looking forward, @vickcoo 😊

**Comment by andrewtavis** [comment]
Hey @vickcoo 👋 Checking in on this with you :) Would you still have interest in working on this feature?

**Comment by vickcoo** [comment]
> Hey @vickcoo 👋 Checking in on this with you :) Would you still have interest in working on this feature?

Hi @andrewtavis, sorry i forgot to solve this issue, i couldn't to work on this, because i am busy recently🥲

**Comment by andrewtavis** [comment]
No stress at all, @vickcoo! Do you think you'll have time to work on it sometime next month? If so we could pick it up then, or I can also unassign it you'd prefer :)
